plugins/BEdita/Core/tests/TestCase/ORM/Inheritance/QueryTest.php

```diff
@@ -219,6 +219,7 @@ public function aliasFieldProvider()
      * @dataProvider aliasFieldProvider
      * @covers ::fixAliasField()
      * @covers ::extractField()
+     * @covers ::aliasChecker()
```

Comment:
For this purpose, the [`@uses`](https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.uses) annotation would probably fit better. However, we don't currently use it — it is meant for very strict test suites in which one is required to explicitely list _all_ the methods the test will flow through. If code executed during a test happens to be not listed amongst the `@uses`, the test [is marked as risky](https://phpunit.de/manual/current/en/risky-tests.html#risky-tests.unintentionally-covered-code). 🙈  It would be clearly unfeasible in our case.